includes/class-wp-job-manager-email-notifications.php

```diff
@@ -933,4 +965,35 @@ private static function get_styles() {
 		return ob_get_clean();
 	}
 
+	/**
+	 * Assemble multipart e-mail body.
+	 *
+	 * @param string $content_html
+	 * @param string $content_plain
+	 *
+	 * @return string
+	 */
+	private static function get_multipart_body( string $content_html, string $content_plain ): string {
+		$multipart_body = '';
+
+		if ( ! empty( $content_html ) ) {
+			$multipart_body .= '
+--' . self::MULTIPART_BOUNDARY . '
```

Comment:
Hey @yscik just noticed that we add the `--` both in here and in the variable.

Reply:
Oops that's not good, I think the Content Type header shouldn't include it. I'll open a PR.